demos/gesture_recognition_demo/cpp_gapi/include/stream_source.hpp

```diff
@@ -147,6 +155,14 @@ class CustomCapSource : public cv::gapi::wip::IStreamSource
 
         /** Put fast frame to the batch **/
         producer.fillFastFrame(fast_frame);
+        if (wait_gap) {
+            const auto cur_step = std::chrono::steady_clock::now() - read_time;
+            const auto gap = std::chrono::duration_cast<std::chrono::milliseconds>(cur_step).count();
+            const int time_step = int(1000.f / float(source_fps));
+            if (gap < time_step) {
+                std::this_thread::sleep_for(std::chrono::milliseconds(time_step - gap));
```

Comment:
I think `sleep_for()` should be able to cast to appropriate duration on its own.

Reply:
`sleep_for()` get `std::chrono::duration<type, per>`. I must wrap `int` value to `std::chrono::milliseconds::duration<int64_t, void>`.

Reply:
I can wrap `time_step` and use `std::this_thread::sleep_for(time_step - gap)`

Reply:
Applied second variant